example_versioned_docs/version-latest/27-url-parsing.md

```diff
@@ -0,0 +1,49 @@
+---
+title: URL parsing
+id: url-parsing
+slug: /url-parsing
+sidebar_label: 27. URL parsing
+description: Parse urls in Wing
+keywords: [Wing language, URL]
+---
+
+Parse urls using the http module, works with inflight closures.
+
+```js playground example title="main.w"
+bring http;
+bring cloud;
+
+new cloud.Function(inflight () => {
+  let parts = http.parseUrl("postgres://user:pass@host.com:5432/path?k=v#f");
```

Comment:
I'm not sure there's reason this needs to be inflight-only. Can you open an issue for supporting both phases?

Reply:
Yeah sounds good to me, I was wondering that myself!

Reply:
https://github.com/winglang/wing/issues/7100

If its small enough, maybe we can pair on it? Would love to understand how to fix this!